server/pkg/item/query.go

```diff
@@ -3,16 +3,14 @@ package item
 import "github.com/reearth/reearth-cms/server/pkg/id"
 
 type Query struct {
-	workspace id.WorkspaceID
-	project   id.ProjectID
-	q         string
+	project id.ProjectID
```

Comment:
Did you update `schemas/integration.yml`?

Reply:
I don't believe that we have a query integration yet